draft-ietf-tls-svcb-ech.md

```diff
@@ -82,7 +82,7 @@ If all HTTPS records for an alt-authority contain "ech" SvcParams, the client MU
 
 A SVCB RRSet containing some RRs with "ech" and some without is vulnerable to a downgrade attack: a network intermediary can block connections to the endpoints that support ECH, causing the client to fall back to a non-ECH endpoint.  This configuration is NOT RECOMMENDED. Zone owners who do use such a mixed configuration SHOULD mark the RRs with "ech" as more preferred (i.e. lower SvcPriority value) than those without, in order to maximize the likelihood that ECH will be used in the absence of an active adversary.
 
-Use of ECH yields an anonymity set of cardinality equal to the number of ECH-enabled server domains supported by a given client-facing server. Thus, even with an encrypted ClientHello, an attacker who can enumerate the set of ECH-enabled domains supported by a client-facing server can guess the correct SNI with probability at least 1/K, where K is the size of this ECH-enabled server anonymity set. This probability may be increased via traffic analysis or other mechanisms.
+Use of ECH yields an anonymity set whose cardinality is at most the number of ECH-enabled server domains supported by a given client-facing server. Thus, even with an encrypted ClientHello, an attacker who can enumerate the set of ECH-enabled domains supported by a client-facing server can guess the correct SNI with probability at least 1/K, where K is the size of this ECH-enabled server anonymity set. This probability may be increased via traffic analysis or other mechanisms.
```

Comment:
I wonder if "at best" or "roughly proportional to" would be better than "at most"?
I'm having trouble understanding exactly what is meant by "at most".
The "traffic analysis and other mechanisms" does help clarify, but it is squishy.

Reply:
OK, I reworked this paragraph to try to make it more intelligible and accurate.  Please review.